run-compose.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+docker pull pgvector/pgvector:pg16
```

Comment:
I guess this is not needed since docker compose will download the image if not found